docs/quickstarts/insights-blueprints/metadata.yml

```
@@ -0,0 +1,6 @@
+kind: QuickStarts # kind must always be "QuickStarts"
+name: insights-blueprint
```

@elianerpereira this name is the important one I think.
```suggestion
name: insights-creating-blueprint-images
```